lib/puppet_litmus.rb

```diff
@@ -7,6 +7,7 @@ module PuppetLitmus; end
 require 'puppet_litmus/inventory_manipulation'
 require 'puppet_litmus/puppet_helpers'
 require 'puppet_litmus/rake_helper'
+require 'litmus_spec_helper_acceptance'
```

Comment:
I think this should be in `puppet_litmus` as well:
```suggestion
require 'puppet_litmus/spec_helper'
```

I'm also wondering if it should be included by default.

Reply:
yeah, that would make more sense